node/src/bin/manager.rs

```diff
@@ -1097,6 +1120,31 @@ async fn main() -> anyhow::Result<()> {
             let (store, primary_pool) = ctx.store_and_primary();
             commands::prune::run(store, primary_pool, deployment, history, prune_ratio).await
         }
+        Drop {
+            deployment,
+            current,
+            pending,
+            used,
+            force,
+        } => {
+            let sender = ctx.notification_sender();
+            let (store, mut pools) = ctx.store_and_pools();
+            let subgraph_store = store.subgraph_store();
+            let primary_pool = pools
+                .remove(&*PRIMARY_SHARD)
+                .expect("there is a primary pool");
```

Comment:
There's a `ctx.store_and_primary()` which will do most of this for you

Reply:
Fixed in 9ccee3a82